### PR TITLE
Refactor qt workflow triggers

### PR DESCRIPTION
**Pull request by birkskyum**
Part of #503, for qt-ci.

### CODE REVIEW

**Comment by birkskyum** [comment]
@ntadej , can you see anything missing, or something that doesn't need to be added?

**Comment by birkskyum** [comment]
it seems like the qt workflows are the only that doesn't run `npm install` or `npm ci`, and thus doesn't need to trigger on changes to package.json or package-lock.json.

File: .github/workflows/qt-ci.yml [hunk 1]
```diff
@@ -7,63 +7,33 @@ on:
...
-      - ".github/workflows/node-release.yml"
-      - ".github/workflows/qt-ci-windows.yml"
-      - ".github/workflows/update-gl-js.yml"
-  
+    paths:
+      - "bin/**"
+      - "expression-test/**"
+      - "include/**"
+      - "metrics/**"
+      - "platform/qt/**"
```
Comment (ntadej) [inline .github/workflows/qt-ci.yml:15]: `platform/default` is used by all (I think)

**Review by ntadej — CHANGES_REQUESTED** [review]

**Comment by ntadej** [comment]
You need to include the workflow itself 😊

**Comment by ntadej** [comment]
Also the linux builder action...

File: .github/workflows/qt-ci.yml [hunk 2]
```diff
@@ -7,63 +7,41 @@ on:
...
-      - ".github/workflows/update-gl-js.yml"
-  
+    paths:
+      - ".github/actions/qt5-build/**"
+      - ".github/actions/qt6-build/**"
+      - ".github/workflows/qt-ci.yml"
+      - "bin/**"
+      - "expression-test/**"
+      - "include/**"
+      - "metrics/**"
```
Comment (ntadej) [inline .github/workflows/qt-ci.yml:17]: I think `maplibre-gl-js` also needs to be included for shaders.

**Review by ntadej — CHANGES_REQUESTED** [review]

File: .github/workflows/qt-ci.yml (hunk 2 above)
Comment (birkskyum) [reply .github/workflows/qt-ci.yml:17]: the .gitmodules is added already - isn't that enough?

Comment (ntadej) [reply .github/workflows/qt-ci.yml:17]: Ah, true 😊

**Review by ntadej — APPROVED** [review]

**Pull request merged**